src/models/components/stochastic_policy/normal_stochastic_policy.py

```diff
@@ -0,0 +1,21 @@
+import torch
+import torch.distributions as distributions
+import torch.nn as nn
+from torch import Tensor
+from torch.distributions.distribution import Distribution
+
+from .stochastic_policy import StochasticPolicy
+
+
+class NormalStochasticPolicy(StochasticPolicy):
+    def __init__(self, dim_input: int, dim_out: int):
+        super().__init__()
+        self.fc_mean = nn.Linear(dim_input, dim_out)
+        self.fc_std = nn.Linear(dim_input, dim_out)
+        self.relu = nn.ReLU()
+
+    def forward(self, input: Tensor) -> Distribution:
+        mean = self.fc_mean(input)
+        std = self.relu(self.fc_std(input)) + 1e-7  # 0だとエラーなる
```

Comment:
`softplus`がよく使われます。コメントに日本語は文字コード問題を起こす恐れがあるのでできれば避けてください。
```suggestion
        std = torch.softplus(self.fc_std(input)) + 1e-7  # Avoid 0 error.
```